proto/meta.proto

```diff
@@ -529,9 +532,33 @@ message TableParallelism {
   }
 }
 
+// Changes a streaming job in place by
+// overwriting the labels of a materialized view/table/sink with the given id
+message UpdateStreamingJobRequest {
+  // Id of the materialized view, table, or sink which we want to update
+  uint32 id = 1;
+
+  // replace the node_labels map of the streaming job with a given id 
+  // with below map
+  map<string, string> node_labels = 2;
+}
+
+// We do not need to add an explicit status field here, we can just use the RPC status
+message UpdateStreamingJobResponse {}
+
+message GetStreamingJobsRequest {}
+
+// Descriptions of MVs and sinks
+message GetStreamingJobResponse {
+  repeated catalog.Table materialized_views = 1;
```

Comment:
Are the rest fields in catalogs necessary for fetching streaming job status? As I mentioned above, I can we can simplify it to return some necessary status for streaming job rather than the whole catalogs. The status can be returned from the streaming job metadata( `TableFragments` or `streaming_job` table). Cc @shanicky

Reply:
Accordingly the naming of `GetStreamingJobs` is too general, I suggest to change it to `GetStreamingJobStatus` etc.

Reply:
How about `GetStreamingJobsStatus` since we get the statuses of all streaming jobs at once?